.golangci.yml

```diff
@@ -0,0 +1,11 @@
+linters:
+  enable:
+    - megacheck
+    - govet
+    - errcheck
+    - unused
+    - structcheck
```

Comment:
Why only this linters?

Reply:
Maybe as first instance, it could be a good idea, but i think it will be better to active all, except `gomnd`  and `gochecknoglobals `

Reply:
But don't worry, I will do it later 😃

Reply:
yes, definitely, we need to add more. 
But code needs to be fixed before so and this PR is already getting longish...

Reply:
Definitely more linters need to be added, I just started with this basic set as this PR is getting longish

Reply:
All Fixed! 🎉